downloaded/installLisk.sh

```diff
@@ -54,6 +54,12 @@ prereq_checks() {
 		echo "Error: tar is not installed. Exiting."
 		exit 2
 	fi
+	POSTGRES_PROCESS_NUM=$(pgrep -c postgres)
+	if [[ $FRESH_INSTALL == 'true' && $POSTGRES_PROCESS_NUM -gt 1 ]] ; then
+		echo "Error: Postgresql is already running"
+		echo "Postgresql by default listens on 127.0.0.1:5432 and attempting to run two instances at the same time will result in this installation failing"
```

Comment:
If the main issue in this regard is, that both postgres instances are using the same port, maybe we could suggest as a possible solution, to change the port of the existing postgres-installation, in order to successfully run the script.